src/content/chapters/02_a_little_logic.mdx

```diff
@@ -0,0 +1,226 @@
+---
+chapter: 2
+title: "A Little Logic"
+---
+import Info from "../../components/Info.astro";
+import Latex from "../../components/Latex.astro";
+
+Logic is the study of correct reasoning. Among other things, it enables us to
+prove, deduce, and solve puzzles.
+
+This chapter is about Logic because Eberban's jargon is based on Logic jargon.
+
+Naturally, this chapter is really jargon-heavy, so it's alright if it takes a
+while to understand it all. The jargon blocks below are the only bits relevant
+to Eberban; the rest is just extra for experts 🙂
+
+## Propositional Logic
+
+<Info type="jargon" title="Propositions">
+A proposition is a statement that is true or false.
```

Comment:
Eberban use trivalent logic: (provably) true, (provably) false and unknown/unproven.

Reply:
I will preface this sentence with `In logic` and will clarify in a later chapter that Eberban uses trivalent logic.